test/Controller/AboutPeopleControllerTest.php

```diff
@@ -16,7 +16,7 @@ public function it_displays_the_people_page()
 
         $crawler = $client->request('GET', $this->getUrl());
 
-        $this->assertSame(200, $client->getResponse()->getStatusCode());
+        $this->assertSame(200, $client->getResponse()->getStatusCode(), $client->getResponse()->getContent());
```

Comment:
also can be reverted, as it's out of sync with all the other assertions and was only introduced for debugging